dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -174,16 +174,22 @@ http.dispatcher=HTTP Dispatcher
 http.dispatcher.desc=HTTP Dispatcher configuration.
 
 dispatcher.trustedHeaderOrigin=Trusted private header origin
-dispatcher.trustedHeaderOrigin.desc=Private headers are used by the web server plug-in to provide information about the original \
- request. These headers take precedence over the http Host header, and are used to select a virtual host to service a request. The \
- default value is '*', which will trust incoming private headers from any source. Specify 'none' to disable private headers and rely \
- only on the http Host header, or specify a list of IP addresses to restrict private header processing to specific trusted sources.
+dispatcher.trustedHeaderOrigin.desc=Private headers are used by the web server plug-in to provide information about the original request. \
```

Comment:
suggested edit:

The web server plug-in uses private headers  to provide information about the original request. These headers take precedence over the HTTP Host header and are used to select a virtual host to service a request. The default value is '*', which trusts incoming private headers from any source. To disable private headers and rely only on the http Host header, specify 'none'. To restrict private header processing to specific trusted sources, specify a comma-separated list of IP addresses and hostnames. This list supports '*' wildcards, with the following restrictions: IP addresses cannot be shortened and must contain a value for each field, for example "127.0.0.*" or "0:0:0:0:0:ffff:*:*", and hostnames must start with "*.", for example "*.ibm.com". The following example shows a valid list that includes wildcards: "localhost, 127.0.0.1, 192.168.*.*, 0:0:0:0:0:ffff:*:*, *.ibm.com".

Reply:
* is rendering as italic due to markdown- ignore italics